packages/core/docs/guide/user-profile.md

```diff
@@ -0,0 +1,203 @@
+# User profile
+
+<!-- TODO: Add links to 'Composition API' and 'Composables' pages when they are ready -->
+
+> This page assumes you're familiar with Composition API and Composables. Read them first if you are new to these concepts.
+
+[[toc]]
+
+## Checking if the user is logged in
+
+Many interactions in the application are only available or look different if the customer is logged in. To check that, we will use [useUser](../composables/use-user.md), which is one of (if not the most) widely used composable in Vue Storefront.
+
+Like with all other composables, it's important to remember to call `load` before accessing any other property or function of `useUser`. Otherwise, `isAuthenticated` will always return `false`.
+
+```js{8,18}
+import { useUser } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      isAuthenticated
+    } = useUser();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      load,
+      isAuthenticated
+    }
+  }
+}
+```
+
+## Loading current user
+
+When you confirmed that the user is logged in, you may safely access user data.
+
+```js{8,18}
+import { useUser } from '{INTEGRATION}';
```

Comment:
I think it's worth keeping, so (as @NataliaTepluhina said) examples can be copy-pasted to the project, especially because in the next example `userGetters` is imported as well.

@NataliaTepluhina ?

Reply:
I agree with @filipsobol here